zilliqa/benches/it.rs

```diff
@@ -66,6 +66,11 @@ fn process_empty(c: &mut Criterion) {
                 consensus.minimum_stake = "1"
                 consensus.eth_block_gas_limit = 1000000000
                 consensus.gas_price = "1"
+                consensus.genesis_fork.at_height = 0
+                consensus.genesis_fork.failed_scilla_call_from_gas_exempt_caller_causes_revert = true
+                consensus.genesis_fork.call_mode_1_sets_caller_to_parent_caller = true
+                consensus.genesis_fork.scilla_messages_can_call_evm_contracts = true
+                consensus.genesis_fork.scilla_contract_creation_increments_account_balance = true
```

Comment:
Sorry - Please can we merge https://github.com/Zilliqa/zq2/pull/2244 instead? These were meant to have defaults.

Reply:
I've removed these now 2244 is merged.